srcpkgs/wpa_supplicant/files/wpa_supplicant/run

```diff
@@ -1,4 +1,13 @@
 #!/bin/sh
 [ -r ./conf ] && . ./conf
+
+if [ -z "${OPTS}" ] && [ -z "${CONF_FILE}" ] && [ -z "${WPA_INTERFACE}" ]; then
+	for f in /etc/wpa_supplicant/wpa_supplicant-*.conf /etc/wpa_supplicant-*.conf; do
+		case "$f" in *"/wpa_supplicant-*.conf") continue ;; esac
+		iface=${f#*/wpa_supplicant-}
+		AUTO="${AUTO} -N -c ${f} -i ${iface%.conf}"
```

Comment:
AUTO created but not used

Reply:
also, -N fails if there is not another interface before or after it. I'm not great with shell but something like this:
```
                if [ ! -z "${AUTO}" ]; then
                        AUTO="${AUTO} -N"
                fi
                AUTO="${AUTO} -c${f} -i${iface%.conf}"
```
works for me, then:
```
OPTS="${AUTO} -s"
```
after the loop. I don't have multiple wireless interfaces to test it on, but adding a dummy interface seemed to pass the syntax test and fail on the interface not existing.